packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -24,6 +24,26 @@ const cli = {
         describe: 'A list of themes for the test run.',
         default: ['terra-default-theme'],
       },
+      hostname: {
+        type: 'string',
+        describe: 'Automation driver host address.',
+      },
+      port: {
+        type: 'number',
+        describe: 'Automation driver port.',
+      },
+      baseUrl: {
+        type: 'string',
+        describe: 'The base url.',
+      },
+      suite: {
+        type: 'array',
+        describe: 'Overrides specs and runs only the defined suites.',
+      },
+      spec: {
```

Comment:
Should this be `specs` with an `s` that matches [here](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/config/wdio.conf.js#L42)?

Reply:
No, this tricked me too when I first added it because it's an `array` type but singular flag. I tested this locally and you can add an array of options:

Checkout the CLI options here:

https://webdriver.io/docs/clioptions.html

Reply:
Ah, I was looking at https://webdriver.io/docs/options.html#specs.